knotx-server/src/main/java/io/knotx/server/KnotxServerVerticle.java

```diff
@@ -133,7 +134,18 @@ public void start(Future<Void> fut) {
     router.route().failureHandler(ErrorHandler.create(configuration.displayExceptionDetails()));
 
     createHttpServer()
-        .requestHandler(router::accept)
+        .requestHandler(request -> {
+          try {
+            router.accept(request);
+          } catch (IllegalArgumentException ex) {
+            HttpResponseStatus status = HttpResponseStatus.BAD_REQUEST;
```

Comment:
`status` might be a constant so we don't allocate a new object here every time (`HttpResponseStatus.BAD_REQUEST` is not an `enum`).

Reply:
done